catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/TestHiveTableOperations.java

```diff
@@ -41,7 +41,7 @@ public class TestHiveTableOperations extends MiniHiveMetastoreService {
   private static Partition existingPartition;
 
   @BeforeAll
-  private static void setup() {
+  public static void setup() {
```

Comment:
Why change the access modifier to `public`?

Reply:
When running related test UT, IDEA has a prompt, so it is modified by the way.

 Here is a non-necessary modification, I can revert.

![image](https://github.com/datastrato/gravitino/assets/3898450/2bfa915b-9f1a-43b3-8ff0-e84a62598910)